content/en/blog/_posts/2024-05-13-Gateway-API-v1.1/index.md

```diff
@@ -0,0 +1,199 @@
+
+The Conformance report API has been expanded with the `mode` field (intended to
+specify the working mode of the implementation), and the `gatewayAPIChannel`
+(standard or experimental). The `gatewayAPIVersion` and `gatewayAPIChannel` are
+now filled in automatically by the suite machinery, along with a brief
+description of the testing outcome. The Reports have been reorganized in a more
+structured way, and the implementations can now add information on how the tests
+have been run and provide reproduction steps.
+
+Leading Contributors: Christine Kim, Mattia Lavacca, Shane Utt
+
+### New Additions to Experimental Channel
+
+#### [Gateway Client Certificate Verification](https://gateway-api.sigs.k8s.io/geps/gep-91/)
+
+Gateways can now configure client cert verification for each Gateway Listener by
+introducing a new field `frontendValidation` field within `tls`. This field
+supports configuring a list of CA Certificates that can be used as a trust
+anchor to validate the certificates presented by the client.
+
+Leading Contributor: Arko Dasgupta
+
+#### [Session Persistence and BackendLBPolicy](https://gateway-api.sigs.k8s.io/geps/gep-1619/)
+
+[Session Persistence](https://gateway-api.sigs.k8s.io/reference/spec/#gateway.networking.k8s.io%2fv1.SessionPersistence) is being introduced to Gateway API via a new policy
+([BackendLBPolicy](https://gateway-api.sigs.k8s.io/reference/spec/#gateway.networking.k8s.io/v1alpha2.BackendLBPolicy))
+for Service-level configuration and as fields within HTTPRoute
+and GRPCRoute for route-level configuration. The BackendLBPolicy and route-level
+APIs provide the same session persistence configuration, including session
+timeouts, session name, session type, and cookie lifetime type.
+
+Leading Contributors: Grant Spence, Gina Yeh
+
+#### [TLS Terminology Clarifications](https://gateway-api.sigs.k8s.io/geps/gep-2907/)
+
+As part of a broader goal of making our TLS terminology more consistent
+throughout the API, we've introduced some breaking changes to BackendTLSPolicy.
+This has resulted in a new API version (v1alpha3) and will require any existing
+users of this policy to uninstall the v1alpha2 version before installing this
+newer version.
+
+Any references to v1alpha2 BackendTLSPolicy fields will need to be updated to
+v1alpha3. Specific changes to fields include:
+- TargetRef is changed to allow multiples: BackendTLSPolicySpec.TargetRef
+becomes a slice named `TargetRefs`.
+- TLS is changed to `Validation`, and the type name changes from
+BackendTLSPolicyConfig to  `BackendTLSPolicyValidation`.
+- CACertRefs becomes `CACertificateRefs` in what is now `BackendTLSPolicyValidation`
+- WellKnownCACerts becomes `WellKnownCACertificates` and the type name
+changes from WellKnownCACertType to `WellKnownCACertificatesType`
+- the constant WellKnownCACertSystem becomes `WellKnownCACertificatesSystem`
```

Comment:
This is written from the perspective of the Go types, but elsewhere we talk from the perspective of the YAML (e.g. the Go field is `TargetRef`, but in YAML that appears as `targetRef`). Someone who is more familiar with this than I am should doublecheck this, but maybe

```suggestion
- `targetRef` becomes `targetRefs` to allow a BackendTLSPolicy to attach to multiple targets;
- `TLS` is changed to `validation`, and its type name changes from
`BackendTLSPolicyConfig` to  `BackendTLSPolicyValidation`;
- `TLS.CACertRefs` becomes `validation.CACertificateRefs`;
- `wellKnownCACerts` becomes `wellKnownCACertificates`, and its type name
changes from `WellKnownCACertType` to `WellKnownCACertificatesType`; and
- The constant `WellKnownCACertSystem` becomes `WellKnownCACertificatesSystem`.
```

?

Reply:
Following the https://kubernetes.io/docs/contribute/style/style-guide/, I use `code style` for field names and CamelCase for objects, type names, and constant names.

Reply:
We write API kinds in UpperCamelCase and field names in the case you would see in JSON within an HTTP message.

Thanks for the update @candita